lib/puppet/file_serving/configuration/parser.rb

```diff
@@ -79,14 +85,20 @@ def deny(mount, value)
         mount.info _("denying %{val} access") % { val: val }
         mount.deny(val)
       rescue Puppet::AuthStoreError => detail
-        raise ArgumentError.new(_("%{detail} in %{file}, line %{line_num}") % { detail: detail.to_s, file: @file, line_num: @count })
+        error_location_str = Puppet::Util::Errors.error_location(@file, @count)
+        raise ArgumentError.new("%{detail} %{error_location}" %
+                                    { detail: detail.to_s, error_location: error_location_str  })
       end
     }
   end
 
   # Create a new mount.
   def newmount(name)
-    raise ArgumentError.new(_("%{mount} is already mounted at %{name} in %{file}, line %{line_num}") % { mount: @mounts[name], name: name, file: @file, line_num: @count }) if @mounts.include?(name)
+    if @mounts.include?(name)
+      error_location_str = Puppet::Util::Errors.error_location(@file, @count)
+      raise ArgumentError.new(_("%{mount} is already mounted at %{name} at %{error_location}") %
```

Comment:
Should this not have an " at " between "%{name}" and "%{error_location}"?  Inconsistent with line 89 above.

Reply:
In this case we actually know the sentence so I've left the 'at' in the sentence to help with readability.